src/Components/3DTiles/3DTilesUtils.js

```diff
@@ -526,19 +535,20 @@ export function getMeshFromTile(tile) {
   }
 
   //Find the 'Mesh' part of the tile
-  while (!!tile.children[0] && !(tile.type === 'Mesh')) {
+  while (!!tile.children[0] && !(tile.children[0].type === 'Mesh')) {
```

Comment:
why double exclamation mark ?

Reply:
"Converts Object to boolean. If it was falsey (e.g. 0, null, undefined, etc.), it will be false, otherwise, true." ([!! operator](https://stackoverflow.com/questions/784929/what-is-the-not-not-operator-in-javascript))